modules/ecr/main.tf

```diff
@@ -0,0 +1,18 @@
+# tfsec:ignore:aws-ecr-enforce-immutable-repository tfsec:ignore:aws-ecr-repository-customer-key
```

Comment:
`tfsec:ignore:aws-ecr-enforce-immutable-repository` is it still valid? I see `image_tag_mutability = "IMMUTABLE"` below

Reply:
I forgot to remove it after changing 🙏🏻 Updated it in 67ecf10 ✅